workstations.yml

```diff
@@ -1,12 +1,11 @@
 ---
-- hosts: clients:mediaboxen
+- hosts: workstations:mediaboxen
   roles:
-    - network
     - xbps-repoconf
     - core-daemons
     - core-programs
     - sshd
-    - client
     - netlogon
     - cups-client
     - netdata
+    - workstation
```

Comment:
What determines what's a role in workstations.yml vs a dependency of the 'workstation' role?  i.e. why is cups-client here and not in /roles/workstation/meta/main.yml?